contracts/contracts/token/WOETH.sol

```diff
@@ -109,31 +109,28 @@ contract WOETH is ERC4626, Governable, Initializable {
 
     // @notice Called to start a yield period, if one is not active
     function startYield() public {
-        // If we are currently giving yield, do not adjust the yield rate
+        // If we are currently distributing yield, continue until done
         if (block.timestamp < yieldEnd) {
             return;
         }
+        // Change to next yield period
+        yieldEnd = (block.timestamp + YIELD_TIME).toUint128();
```

Comment:
doesn't this extend the yield dripping even when there is no yield to be dripped out? For example in case of `_actualAssets <= _computedAssets `